.cargo-husky/hooks/pre-commit

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+set -e
+
+#echo '+cargo clippy -- -D warnings'
```

Comment:
Forget uncomment?

Reply:
You're right :facepalm: That was failing in my machine, let me have a look at that and I'll get back to you.